### PR TITLE
Adding tgui formula (stable)

### PR DESCRIPTION
**Pull request by coldeasy**

### CODE REVIEW

**Comment by coldeasy** [comment]
As per comment in https://github.com/Homebrew/homebrew/pull/45991

**Comment by apjanke** [comment]
audit failure:
```
==> audit problems
tgui:
 * Stable: version 0.6.9 is redundant with version scanned from URL
```

Just remove the `version` line.

**Comment by coldeasy** [comment]
@apjanke updated

File: Library/Formula/tgui.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Tgui < Formula
+  desc "GUI library for use with sfml"
+  homepage "http://tgui.eu"
```
Comment (bfontaine) [inline Library/Formula/tgui.rb]: You can use `https` here.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @coldeasy; we appreciate it! :smile:

**Pull request closed without merging**